test/functional/contested_test.js

```diff
@@ -231,15 +227,6 @@ Scenario('Contested Matrimonial Case Creation by Solicitor @nightly', async I =>
     await I.contestedOtherDocuments();
     await I.contestedCheckYourAnswers('Matrimonial');
     I.waitForText('Form A Application', '60')
-    I.contestedAmendApplicationDetails();
-    await I.caseSubmitAuthorisation('contested');
```

Comment:
Whats reason these lines are deleted, do we have new test to cover removed lines?

Reply:
yes I have divided this test so I will submit case and amend in another scenario (this is in my TODO list but I have not pushed that commented code in master as it will look nice so keeping it in nightly dev for now). Too many UI calls makes test too long to run so I am distributing few tests and will use API calls to excuse it fast.